components/__init__.py

```diff
@@ -112,22 +112,21 @@ def define_mercury(reqs):
                           'make install'], libs=['opa'],
                 package='openpa-devel' if inst(reqs, 'openpa') else None)
 
-    if ARM_PLATFORM:
-        url = "https://github.com/mercury-hpc/mchecksum.git"
-        retriever = GitRepoRetriever(url)
-        reqs.define('mchecksum',
-                    retriever=retriever,
-                    commands=['cmake -DBUILD_SHARED_LIBS=ON $MCHECKSUM_SRC'
-                              '-DBUILD_TESTING=ON '
-                              '-DCMAKE_INSTALL_PREFIX=$MCHECKSUM_PREFIX '
-                              '-DMCHECKSUM_ENABLE_COVERAGE=OFF '
-                              '-DMCHECKSUM_ENABLE_VERBOSE_ERROR=ON '
-                              '-DMCHECKSUM_USE_ZLIB=OFF '
-                              '-DCMAKE_INSTALL_RPATH=$MCHECKSUM_PREFIX/lib '
-                              '-DCMAKE_INSTALL_RPATH_USE_LINK_PATH=TRUE ',
-                              'make $JOBS_OPT', 'make install'],
-                    libs=['mchecksum'],
-                    out_of_src_build=True)
+    url = "https://github.com/mercury-hpc/mchecksum.git"
```

Comment:
We don't currently use mchecksum on x86_64.  Is something changing?

Reply:
I don't think we actually need it, but I was including the component definition here so it was available for a tool I created that benchmarks the different libraries and checksums. I didn't think it would hurt anything having the component defined here if it isn't included in the required prereqs in the Sconscript for the daos libraries. But I can easily remove it if it makes things too convoluted.

Reply:
If it's only used by your tool, you can add a define in your SConstruct (see daos SConstruct line 47-48 which define cmocka and readline).   But there is probably no harm in having it here either.